module-db/Tables/MultimediaFilesTable.hpp

```diff
@@ -70,6 +70,12 @@ namespace db::multimedia_files
         channels
     };
 
+    enum class SortingBy
+    {
+        Title,
```

Comment:
I'd change those to `TitleAscending` and `IdAscending` as it's more precise and won't need changing in case descending sorting order is added one day :wink:

Reply:
done